gateway-discovery-cm/src/test/java/org/apache/knox/gateway/topology/discovery/cm/ClouderaManagerServiceDiscoveryTest.java

```diff
@@ -988,8 +999,17 @@ private void doTestSolrDiscovery(final boolean isSSL) {
   private ServiceDiscovery.Cluster doTestLivyDiscovery(final String  hostName,
                                                        final String  port,
                                                        final boolean isSSL) {
+    return doTestLivyDiscovery(hostName, port, isSSL, "http");
+  }
+
+
+  private ServiceDiscovery.Cluster doTestLivyDiscovery(final String  hostName,
+                                                       final String  port,
+                                                       final Boolean isSSL,
+                                                       final String  transportMode) {
     // Configure the role
     Map<String, String> roleProperties = new HashMap<>();
+    roleProperties.put("livy_server_thrift_transport_mode", transportMode);
```

Comment:
We shouldn't care about this. Livy has a thrift server which is completely separate from the HTTP API.

Reply:
Understood